docs/kb/params.md

```diff
@@ -0,0 +1,16 @@
+---
+order: 15
+description: Module Parameters
+---
+
+# Module Parameters
+
+Sometimes the developer needs to set some default parameters for a module. The [cosmos-sdk params package](https://docs.cosmos.network/master/modules/params) provides a globally available parameter saved into the store.
+The starport can scaffold parameters to be accessible for the module. These parameters have default values but can change along the chain is alive.
```

Comment:
>change along the chain is alive.

That's the main utility of params, since they are managed and centralized by the `params` SDK module, they can be easily updated through a Governance proposal, I think we should add this statement here